### DISCUSSION TITLE
can you make it work on all tab ?

### DISCUSSION THREAD

**Discussion by nhok0169** [nutti/Screencast-Keys#136] [Ideas]
everytime im change to another  tab like UV editor, layout, script,..
i need turn it on again
can you make it turn on in all tab ?
or it a new error on blender 3.x ?

**Comment by nutti** [comment]
@nhok0169 

Did you try setting **Origin** option to "Window"?

Different workspace handle as different region or area.
So, you can solve your issue by set **Origin** option to "Window".

**Reply by nhok0169** [reply]
thank you
solved
